apps/examples/wc-react18/src/App.js

```diff
@@ -1,33 +1,22 @@
-import React, { useState } from 'react';
-import { createComponent } from '@lit-labs/react';
-import { PieButton } from '@justeattakeaway/pie-button';
-import './styles.css';
+import { useState } from 'react';
+import { PieButton } from '@justeattakeaway/pie-button/dist/react';
 
-// eslint-disable-next-line no-unused-vars
-const PieBtn = createComponent({
-    tagName: 'pie-button',
-    elementClass: PieButton,
-    react: React,
-    events: { onCustomEvent: 'CustomEvent' },
-});
+import './styles.css';
 
 function App () {
     const [counter, setCounter] = useState(0);
 
```

Comment:
I believe there was a PR recently to remove customEvent from `pie-button`, since it could actually just be handled directly inside the consuming apps - https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-button/README.md?plain=1#L68

I tested it by adding the event locally to `onClick` like: `<pie-button onclick="e => console.log(e)">Click me!</pie-button>` and it all works 👍